lib/ansible/template/__init__.py

```diff
@@ -412,9 +412,30 @@ def __init__(self, delegatee, pluginloader, *args, **kwargs):
 
         self._collection_jinja_func_cache = {}
 
+        self._ansible_plugins_loaded = False
+
+    def _load_ansible_plugins(self):
+        if self._ansible_plugins_loaded:
+            return
+
+        for plugin in self._pluginloader.all():
+            method_map = getattr(plugin, self._method_map_name)
+            self._delegatee.update(method_map())
+
+        if self._pluginloader.class_name == 'FilterModule':
+            for plugin_name, plugin in self._delegatee.items():
+                if USE_JINJA2_NATIVE and plugin_name in C.STRING_TYPE_FILTERS:
+                    self._delegatee[plugin_name] = _wrap_native_text(plugin)
+                else:
+                    self._delegatee[plugin_name] = _unroll_iterator(plugin)
```

Comment:
Just because I like to comment on places where I wasn't sure about something. This confused me at first. But since the filters listed in `STRING_TYPE_FILTERS` should never produce a generator or other iterator, we need not wrap them with `_unroll_iterator` also.